### ISSUE TITLE
Lots of bugs in Emesene 2

### ISSUE THREAD

**Issue by pedrofleck** [emesene/emesene#875]
Emesene 2 is really giving me a headache, it have a lot of bugs...
When someone request a videocall, even if I reject it, Emesene don't deliver my messages anymore, for anyone, and I have to reconnect; Sometimes it just don't receive, or send a message, and the conversation turns really confusing, or sometimes I receive the message, but it just appears in the chat history, really strange...; When someone sends me a image, it appears: File transfer invitation, but nothing appears in the window, and when someone send me an archive that appear in the conversation, I click to accept, but the progress bar don't work; If I'm writing to someone, and other person talks to me, I continue writing in the other conversation, but I can't see this, It's REALLY annoying, a lot of times I sent messages to the wrong person...
I love Emesene, I think it's a way better than Windows Live Messenger, or aMSN, but this bugs in the version 2 are driving me crazy...
PS: Sorry my english, I'm brazillian

**Comment by Sbte** [comment]
Video calls are not available in the latest development version or in any stable releases that I know of. What version are you using?

**Comment by pedrofleck** [comment]
2.11.7, Windows version, I know it aren't avaiable, but when someone invites me, appears a window saying that I can see the other, but I won't appear, something like that...

**Comment by pedrofleck** [comment]
I thought that "close" will close the window, not the the threat... haha

**Comment by pepeleproso** [comment]
I can confirm this is happening in git master with emesene->WLM2009
If you open a new video call from WLM2009 papyon goes crazy

**Comment by c10ud** [comment]
which way it goes crazy?

**Comment by lainiwaku** [comment]
"Sometimes it just don't receive, or send a message, and the conversation turns really confusing, or sometimes I receive the message, but it just appears in the chat history, really strange...;"

i have this bug too and  i have  this bug on amsn too 
i think it's an protocole bug since  the news msn 2011 !

**Comment by Sbte** [comment]
And why didn't this disable it?

https://github.com/emesene/emesene/commit/cccf00db34081406ba55beb984893ea61300d7b3

**Comment by c10ud** [comment]
sbte, there are hard "return"s in handlers so it cannot start

**Comment by Sbte** [comment]
c10ud, this is about emesene having capabilities that tell WLM that video is possible. We have to tell WLM that it's not. And that's what i tried to do with that commit

**Comment by c10ud** [comment]
papyon/profile.py:
        self._client_capabilities.supports_sip_invite = True
        self._client_capabilities.supports_tunneled_sip = True

i think you should set those as false like:
https://github.com/emesene/emesene/blob/cccf00db34081406ba55beb984893ea61300d7b3/emesene/e3/papylib/Worker.py#L152

**Comment by c10ud** [comment]
we should set (or "force"):
            'has_webcam': 0x00000010, FALSE
            'has_onecare': 0x01000000, FALSE

```
        'renders_gif': 0x00000004, TRUE
        'renders_isf': 0x00000008, FALSE

        'supports_winks': 0x00008000, FALSE
        'supports_shared_search': 0x00010000, FALSE
        'supports_voice_im': 0x00040000, FALSE
        'supports_sip_invite': 0x00100000, FALSE
        'supports_tunneled_sip': 0x00200000, FALSE
        'supports_shared_drive': 0x00400000, FALSE
        'supports_rtc_video': 0x00000010, FALSE
```

**Comment by Sbte** [comment]
It already is like that by default.

**Comment by c10ud** [comment]
Fixed 050f5f7edc3d940b029a9506e6505969dbb24233
